WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -204,7 +204,7 @@ extension WPStyleGuide {
 
     @objc public class func applyReaderStreamHeaderTitleStyle(_ label: UILabel, usesNewStyle: Bool = false) {
         if usesNewStyle {
-            label.font = WPStyleGuide.serifFontForTextStyle(.title1, fontWeight: .semibold)
+            label.font = WPStyleGuide.fontForTextStyle(.title1, fontWeight: .bold)
```

Comment:
In Figma, this font is `bold` but there's also text next to it indicating that it should be `semibold` (zQnohyMpLzBzQ5jzMMKni3-fi-937:35928).

@osullivanchris should we be using the red text next to the label or what it's actually set to in Figma?